radixdlt/src/main/java/com/radixdlt/StateComputerModule.java

```diff
@@ -79,10 +81,19 @@ public StateComputerModule(int fixedNodeCount, long viewsPerEpoch) {
 	@Override
 	protected void configure() {
 		bind(new TypeLiteral<EngineStore<CommittedAtom>>() { }).to(CommittedAtomsStore.class).in(Scopes.SINGLETON);
-		bind(AtomToBinaryConverter.class).toInstance(new AtomToBinaryConverter(DefaultSerialization.getInstance()));
 		bind(StateComputer.class).to(RadixEngineStateComputer.class);
 	}
 
+	@Provides
+	private CommandToBinaryConverter commandToBinaryConverter(Serialization serialization) {
+		return new CommandToBinaryConverter(serialization);
+	}
+
+	@Provides
+	private ClientAtomToBinaryConverter clientAtomToBinaryConverter(Serialization serialization) {
```

Comment:
These providers are unnecessary.  They only use the passed-in arguments.  Just annotate the constructor with Inject.